Community/Tdarr_Plugin_2sus_Burn_in_Subtitles_Handbrake.js

```diff
@@ -0,0 +1,172 @@
+    Description: 'This plugin can burn in subtitles based on audio track language \n\n',
+    Version: "0.01",
+    Tags: "pre-processing,handbrake,subtitles only,configurable",
+    Inputs: [
+        {
+            name: 'handbrake_preset',
+            type: 'string',
+            defaultValue: 'Fast 1080p30',
+            inputUI: {
+                type: 'text',
+            },
+            tooltip: `Name of the handbrake preset that will be used. \n\n
+            
+            \\nCommon Presets:
+            \\nHQ 1080p30 Surround     if you want surround
+            \\nH.264 MKV 1080p30       if you want mkv
+            \\nFast 1080p30            if you want mp4
+            `
+        },
+        {
+            name: 'extra_args',
+            type: 'string',
+            defaultValue: '',
+            inputUI: {
+                type: 'text'
+            },
+            tooltip: `Extra handbrake args to add to the end of the command.`
+        },
+        {
+            name: 'output_container',
+            type: 'string',
+            defaultValue: '.mp4',
+            inputUI: {
+                type: 'text'
+            },
+            tooltip: `Container the output file will be in. Leave blank to leave container unchanged\n\n
+            
+            Common Containers:
+            mkv
+            mp4
+            `
+        },
+        {
+            name: 'audio_lang',
+            type: 'string',
+            defaultValue: 'jpn',
+            inputUI: {
+                type: 'text',
+            },
+            tooltip: `Specify the audio language subs should be burned in for. If ONLY this language is present, burn in subs.`,
+        },
+        {
+            name: 'sub_lang',
+            type: 'string',
+            defaultValue: 'eng,und',
+            inputUI: {
+                type: 'text',
+            },
+            tooltip: `Specify the language of subtitles to burn in. If a comma separated list, burn in the first matching subtitle\n\n
+            
+            Example: eng,und    burn in first matching eng subs. If none, burn in first matching und subs
+            `,
+        },
+    ],
+});
+
+// eslint-disable-next-line no-unused-vars
+const plugin = (file, librarySettings, inputs, otherArguments) => {
+
+    const lib = require('../methods/lib')();
+    // eslint-disable-next-line no-unused-vars,no-param-reassign
+    inputs = lib.loadDefaultValues(inputs, details);
+    //Must return this object
+
+    var response = {
+        processFile: false,
+        preset: "",
+        container: `${file.container}`,
+        handBrakeMode: true,
+        FFmpegMode: false,
+        reQueueAfter: false,
+        infoLog: "",
+    };
+
+    if (
+        inputs.handbrake_preset === undefined ||
+        inputs.audio_lang === undefined ||
+        inputs.sub_lang === undefined
+    ) {
+        response.processFile = false;
+        response.infoLog += "☒ Inputs not entered! \n";
+        return response;
+    }
```

Comment:
Don't need this due to the default values

Reply:
Removed this block